data/transactions/transaction.go

```diff
@@ -181,6 +182,13 @@ func (tx Transaction) ID() Txid {
 	return Txid(crypto.Hash(enc))
 }
 
+// IDSha256 returns the Txid (i.e., hash) of the transaction.
+func (tx Transaction) IDSha256() Txid {
```

Comment:
conceptually, I think this is a little bit misleading.
we may want to keep the term 'Txid' to be only Sha512_256

this thing should only return a digest